utilities.tex

```diff
@@ -124,13 +118,69 @@
 \setcounter{subsection}{1}
 \rSec2[utility.swap]{swap}
 
-\ednote{The following edits to the two \tcode{swap} overloads are to be applied to the versions in
-namespace \tcode{std}.}
-
+{\color{newclr}
 \indexlibrary{\idxcode{swap}}%
+\pnum The name \tcode{swap} denotes a customization point object
+(\ref{customization.point.object}). The expression
+\tcode{ranges::swap(E1, E2)} for subexpressions \tcode{E1}
+and \tcode{E2} has the same effects as:
+
+\begin{itemize}
+\item
+  \tcode{(void)swap(E1, E2)} if that is a valid expression when
+  overload resolution is performed in a context that includes
+  the declarations
+\begin{codeblock}
+  template <class T>
+  void swap(T&, T&) = delete;
+  template <class T, size_t N>
+  void swap(T(&)[N], T(&)[N]) = delete;
```

Comment:
I think the purpose of these `= delete` poison pill declarations is obscure enough to get a Note.